Gems/ImGui/Code/Source/LYCommonMenu/ImGuiLYCommonMenu.cpp

```diff
@@ -275,12 +276,22 @@ namespace ImGui
 
                         AZ::Data::AssetType levelAssetType = lvlSystem->GetLevelAssetType();
                         AZStd::vector<AZStd::string> levelNames;
+                        AZStd::set<AZStd::string> networkedLevelNames;
                         auto enumerateCB =
-                            [levelAssetType, &levelNames]([[maybe_unused]] const AZ::Data::AssetId id, const AZ::Data::AssetInfo& assetInfo)
+                            [levelAssetType, &levelNames, &networkedLevelNames]([[maybe_unused]] const AZ::Data::AssetId id, const AZ::Data::AssetInfo& assetInfo)
                         {
                             if (assetInfo.m_assetType == levelAssetType)
                             {
-                                levelNames.emplace_back(assetInfo.m_relativePath);
+                                if (assetInfo.m_relativePath.ends_with(".network.spawnable"))
```

Comment:
is there any risk of this changing? Should we move to named const string?

Reply:
There is a const for this specifically in the Multiplayer Gem. Short of a mechanism for Multiplayer to register and provide filtering logic, I think a const string with a comment referring to MultiplayerConstants.h makes sense.

Reply:
thank you :) fixed!